frameworks/Crystal/amber/shard.yml

```diff
@@ -0,0 +1,19 @@
+name: amber
+version: 0.1.0
+
+authors:
+  - Amber <contact@ambercr.io>
+
+license: MIT
+
+dependencies:
+  amber:
+    github: Amber-Crystal/amber 
+    version: ~> 0.1.4
+
+  granite_orm:
+    github: Amber-Crystal/granite-orm
+    version: ~> 0.6.2
```

Comment:
@drujensen I just see this PR https://github.com/Amber-Crystal/granite-orm/pull/39 :+1: 

Do I need to change version to master? or a new version of granite-orm will be released?